jest.config.js

```diff
@@ -0,0 +1,23 @@
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+})
+
+// Add any custom config to be passed to Jest
+const customJestConfig = {
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  moduleNameMapper: {
+    // Handle module aliases (this will be automatically configured for you soon)
```

Comment:
I found the comments helpful as I'm also learning this as I review, but I do know we don't normally have comments. Is that something we can push initially to help each other and then remove before the final approval?

Reply:
Sure, comments are fine if they're helpful but in this case, I think they were copied and pasted and might contain potentially confusing text like "(this will be automatically configured for you soon)".

Reply:
I can definitely do that. Especially since I have no context for it, it's not helpful at all. DELETE!

Reply:
Great! In the future, I'd recommend making change requests in a single commit and copying that SHA into GitHub so the reviewer (me) can easily see these changes being made before considering the conversation resolved.

<img width="1312" alt="image" src="https://user-images.githubusercontent.com/7015466/194611420-531e228b-d809-4da1-a807-61a6883601c3.png">